README.md

```diff
@@ -137,7 +147,7 @@ All components can be imported from `clerk-sveltekit/client/ComponentName.svelte
 - `<ClerkLoaded let:clerk />` — Wrapper that shows its contents (and exposes the `clerk` object) when Clerk is done loading.
 - `<SignIn />` — Renders a sign-in form.
 - `<SignUp />` — Renders a sign-up form.
-- `<SignedIn let:user />` — Wrapper that shows its contents (and exposes the Clerk `user` object) when the user is signed in.
+- `<SignedIn let:user />` — Wrapper that shows its contents when the user is signed in.
```

Comment:
`let:user` should probably be removed because the `user` object isn't exposed anymore

Reply:
Thanks!